html_css/v2/main-course/browser-compatibility.md

```diff
@@ -0,0 +1,70 @@
+
+### Learning Outcomes
+By the end of this lesson, you should:
+
+- Understand browser compatibility and its history.
+- Know how new CSS features make their way into browsers.
+- Know how to check for compatibility.
+
+### Browser History
+The history of modern browsing began back in December of 1990 with the release of WorldWideWeb browser. It was written by Tim Berners-Lee while working for the European nuclear research agency known as CERN. It was later renamed to Nexus, to avoid confusion with the World Wide Web. 
+
+Nexus was the first of its kind, and allowed users to view basic style sheets, read newsgroups and even had spellcheck! It might not seem like a lot today, but at that time it was truly groundbreaking. 
+
+The release of Nexus was just the beginning though, as in the next decade people witnessed the first releases of browsers such as Mosaic Browser, which quickly gained popularity and quickly became the most popular browser on the globe. From there, the growth of the World Wide Web exploded with the releases of Opera and Netscape Navigator browsers. 
+
+In 1995 the world got introduced to the first version of Internet Explorer, which became the dominant player in the market. At some point, Internet Explorer was used by more than 90% of all users. To counter this dominance, Netscape launched what would become Mozilla Foundation which develops and maintains the Firefox. Soon after that, in 2003, Apple launched Safari, and in 2008, Google launched Chrome. 
+
+You're most likely familiar with most, if not all these names. There is a lot of competition among the browsers still to this day, even though Chrome (and chromium) is the dominant player in the market.
+
+### What is Browser Compatibility
+Today, it is impossible to imagine the Web without a use of the browsers. We have witnessed a shift from standalone applications to HTML5 and Progressive Web Apps that allow applications to fully function within a browser. For example, Microsoft Word and Excel for the longest time could only be executed via a standalone application. Now, you can utilize those applications through any browsers without a need to install any files. 
+
+As companies compete for the market share, different browsers are using different engines to display information on the web page. For example, Chrome and chromium utilize Blink, while Safari uses its proprietary WebKit.
+
+Because of these differences, your applications may behave differently in the browser. Due to Chrome dominance, vast majority of applications are designed to work smooth with chromium and providing as good of performance in other browsers is secondary. 
```

Comment:
```suggestion
Because of these differences, your applications may behave differently in the browser. Due to Chrome dominance, the vast majority of applications are designed to work smoothly with Chromium, and providing as good of performance in other browsers is secondary. 
```